core/new-gui/src/app/workspace/component/operator-panel/operator-panel.component.ts

```diff
@@ -61,6 +79,55 @@ export class OperatorPanelComponent implements OnInit {
     this.operatorSchemaList = operatorMetadata.operators;
     this.groupNamesOrdered = getGroupNamesSorted(operatorMetadata.groups);
     this.operatorGroupMap = getOperatorGroupMap(operatorMetadata);
+
+
+
+    this.filteredOptions = this.myControl.valueChanges
+      .map(value => this._filter(value));
+    }
+
+  private _filter(value: string): string[] {
+
+    type userFriendlyNamesType = {
+      title: string;
+    };
+
+    const userFriendlyNames = this.operatorSchemaList.map(value => value.additionalMetadata.userFriendlyName);
+
+    const new_userFriendlyNames : userFriendlyNamesType[] = []
+    for (let entry of userFriendlyNames) {
+      new_userFriendlyNames.push({'title':entry})
+    }
+
+    const filterValue = value.toLowerCase();
+
+    const options: Fuse.FuseOptions<userFriendlyNamesType> = {
+      location:0,
+      distance:1000,
+      threshold: 0.3,
+      keys: ['title']
+    };
```

Comment:
Please add some comments to describe what does each of this parameter do and how will they affect the fuzzy search result.

In addition, I believe `distance` parameter is too big since that is 1000 characters for matching in fuzzy search. I believe 10 might be a decent choice.